### ISSUE TITLE
NullPointerException in GhprbSimpleStatus.onBuildTriggered

### ISSUE THREAD

**Issue by farcaller** [janinko/ghprb#281]
Triggering the hook results in exception for me:

```
Jun 03 21:13:23 ci java[1783]: Caused by: java.lang.NullPointerException
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.extensions.status.GhprbSimpleStatus.onBuildTriggered(GhprbSimpleStatus.java:61)
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.GhprbBuilds.build(GhprbBuilds.java:44)
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.GhprbPullRequest.build(GhprbPullRequest.java:277)
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.GhprbPullRequest.tryBuild(GhprbPullRequest.java:269)
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.GhprbPullRequest.check(GhprbPullRequest.java:174)
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.GhprbRepository.onPullRequestHook(GhprbRepository.java:293)
Jun 03 21:13:23 ci java[1783]: at org.jenkinsci.plugins.ghprb.GhprbRootAction.doIndex(GhprbRootAction.java:107)
```

**Comment by DavidTanner** [comment]
Which version are you using?

**Comment by DavidTanner** [comment]
Should be fixed in 1.22.4.  I am assuming that at that point the repository is null, so I am forcing an init if that is the case.
